src/io/dirops.c

```diff
@@ -33,79 +33,33 @@ static char * UnicodeToUTF8(const wchar_t *str)
 
      return result;
 }
+#endif
 
-static int mkdir_p(MVMThreadContext *tc, wchar_t *pathname, MVMint64 mode) {
-    wchar_t *p = pathname, ch;
-#else
 static int mkdir_p(MVMThreadContext *tc, char *pathname, MVMint64 mode) {
     char *p = pathname, ch;
     uv_fs_t req;
     int mkdir_error = 0;
-#endif
     int created = 0;
 
     for (;; ++p)
         if (!*p || IS_SLASH(*p)) {
             ch = *p;
             *p  = '\0';
-#ifdef _WIN32
-            created = CreateDirectoryW(pathname, NULL);
-#else
             created = (mkdir(pathname, mode) == 0
                        || ( (mkdir_error = errno) == EEXIST
                        && uv_fs_stat(NULL, &req, pathname, NULL) == 0
-                       && S_ISDIR(req.statbuf.st_mode)));
```

Comment:
I deleted the `S_ISDIR` line because it doesn't seem to be available on Windows as-is. Maybe it needs a header included?